### DISCUSSION TITLE
Redirect logs from containers to k3s console

### DISCUSSION THREAD

**Discussion by bademux** [k3s-io/k3s#5530] [Q&A] [answered]
Hi, can I kindly ask for help with the problem below:
I'm using k3s to test my application with to containers in pod. 
Is it possible to stream logs from  containers into the same stdout as k3s, without running kubectl logs or additional k8s api call.
Currently on k3s stdout only k8s events without containers logs.

Thanks!

**Comment by brandond** [comment, marked as answer]
No, it is not possible to have the k3s stdout/stderr also capture the output of pods run by the kubelet. That functionality is not available from the kubelet or containerd. If you're running a single-node cluster, the easiest way to get the logs might be to just tail the files in /var/log/pods.

**Reply by bademux** [reply]
Thanks for the answer!
Since I use testcontainers k3s module it pulls logs from k3s container (I assume), is there way to somehow redirect logs that supposed to land in /var/log/pods into k3s docker container log stream. 
Sorry for naive assumptions, I'm not sure I understand how pod logs are managed.

**Reply by brandond** [reply]
No, there is no way to do that.
